docs/content/en/docs/workshops/introduction/overview.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Overview"
+linkTitle: "Overview"
+weight: 1
+date: 2021-11-11
+description: >  
+---
+
+## What is the purpose of this workshop?
+
+The purpose of this workshop is to provide a more perscriptive walkthrough of building, deploying, and operating an EKS Anywhere cluster. This will use existing content from the documentation, just in a more condensed format for those wishing to get started. 
```

Comment:
```suggestion
The purpose of this workshop is to provide a more prescriptive walkthrough of building, deploying, and operating an EKS Anywhere cluster. This will use existing content from the documentation, just in a more condensed format for those wishing to get started. 
```